Source/Core/VideoCommon/OnScreenDisplay.cpp

```diff
@@ -56,7 +61,7 @@ void DrawMessages()
 
 		alpha <<= 24;
 
-		g_renderer->RenderText(it->str, left, top, 0xffff30 | alpha);
+		g_renderer->RenderText(it->m_str, left, top, it->m_rgb | alpha);
```

Comment:
(it->m_rgb & 0xffffff) | ((it->m_rgb >> 24) * alpha)

and alpha goes from 1.0 to 0.0